docs/source/testing/testing.mdx

```diff
@@ -40,39 +43,52 @@ const resolvers = {
 it('returns hello with the provided name', async () => {
   const testServer = new ApolloServer({
     typeDefs,
-    resolvers
+    resolvers,
   });
 
-  const result = await testServer.executeOperation({
+  const response = await testServer.executeOperation({
     query: 'query SayHelloWorld($name: String) { hello(name: $name) }',
     variables: { name: 'world' },
   });
 
-  expect(result.errors).toBeUndefined();
-  expect(result.data?.hello).toBe('Hello world!');
+  expect(response.result.errors).toBeUndefined();
+  expect(response.result.data?.hello).toBe('Hello world!');
 });
 ```
 
 </MultiCodeBlock>
 
 Note that when testing, any errors in parsing, validating, and executing your GraphQL operation are returned in the `errors` field of the operation result. As with any GraphQL response, these errors are not _thrown_.
 
-Unlike with a normal instance of `ApolloServer`,  you don't need to call [`start`](../api/apollo-server/#start) before calling `executeOperation`. The server instance calls `start` automatically for you if it hasn't been called already, and any startup errors are thrown.
+Unlike with a normal instance of `ApolloServer`, you don't need to start your server (e.g., using `startStandaloneServer` or `expressMiddleware`) before calling `executeOperation`. The server instance will start automatically and throw any startup errors.
```

Comment:
expressMiddleware doesn't start, actually; you need to start before using it. (startStandaloneServer does though)

Reply:
Oh, that's good to know—thank you! I'll correct this